tensorflow/buildspec-2-9.yml

```diff
@@ -41,149 +41,149 @@ context:
-  #   target: sagemaker
-  #   enable_test_promotion: true
-  #   context:
-  #     <<: *TRAINING_CONTEXT
-  # BuildTensorflowSageMakerCpuPy39TrainingDockerImage:
-  #   <<: *TRAINING_REPOSITORY
-  #   build: &TENSORFLOW_CPU_TRAINING_PY3 false
-  #   image_size_baseline: &IMAGE_SIZE_BASELINE 7500
-  #   device_type: &DEVICE_TYPE cpu
-  #   python_version: &DOCKER_PYTHON_VERSION py3
-  #   tag_python_version: &TAG_PYTHON_VERSION py39
-  #   os_version: &OS_VERSION ubuntu20.04
-  #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-sagemaker"
-  #   ]
-  #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.,
-  #                        *DEVICE_TYPE ]
-  #   target: sagemaker
-  #   enable_test_promotion: true
-  #   context:
-  #     <<: *TRAINING_CONTEXT
-  BuildEC2TensorflowCPUInferencePy3DockerImage:
-    <<: *INFERENCE_REPOSITORY
-    build: &TENSORFLOW_CPU_INFERENCE_PY3 false
-    image_size_baseline: &IMAGE_SIZE_BASELINE 4899
-    framework_version: &FRAMEWORK_VERSION 2.9.3
+  BuildTensorflowEC2CpuPy39TrainingDockerImage:
+    <<: *TRAINING_REPOSITORY
+    build: &TENSORFLOW_CPU_TRAINING_PY3 false
+    image_size_baseline: &IMAGE_SIZE_BASELINE 4489
     device_type: &DEVICE_TYPE cpu
     python_version: &DOCKER_PYTHON_VERSION py3
     tag_python_version: &TAG_PYTHON_VERSION py39
     os_version: &OS_VERSION ubuntu20.04
-    tag: !join [ *FRAMEWORK_VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-ec2" ]
-    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-ec2"
+      ]
+    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile.,
+      *DEVICE_TYPE ]
     target: ec2
     enable_test_promotion: true
     context:
-      <<: *INFERENCE_CONTEXT
-  BuildEC2TensorflowGPUInferencePy3DockerImage:
-    <<: *INFERENCE_REPOSITORY
-    build: &TENSORFLOW_GPU_INFERENCE_PY3 false
-    image_size_baseline: &IMAGE_SIZE_BASELINE 7738
-    framework_version: &FRAMEWORK_VERSION 2.9.3
+      <<: *TRAINING_CONTEXT
+  BuildTensorflowEC2GpuCu112Py39TrainingDockerImage:
+    <<: *TRAINING_REPOSITORY
+    build: &TENSORFLOW_GPU_TRAINING_PY3 false
+    image_size_baseline: &IMAGE_SIZE_BASELINE 9307
     device_type: &DEVICE_TYPE gpu
     python_version: &DOCKER_PYTHON_VERSION py3
     tag_python_version: &TAG_PYTHON_VERSION py39
     cuda_version: &CUDA_VERSION cu112
     os_version: &OS_VERSION ubuntu20.04
-    tag: !join [ *FRAMEWORK_VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION, "-ec2" ]
-    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION,
+      "-", *OS_VERSION, "-ec2" ]
```

Comment:
nit: Was this whitespace intentional?

Reply:
I formatted the YML file before uploading